README.md

```diff
@@ -6,76 +6,215 @@
+PHP, Dart, Elixir, Elm, Go, Rust, Java and .NET. It can also update git submodules, Docker files, and Terraform files.
+Features include:
+
+- Check for the latest version of a dependency *that's resolvable given a project's other dependencies*
+- Generate updated manifest and lockfiles for a new dependency version
+- Generate PR descriptions that include the updated dependency's changelogs, release notes, and commits
+
+# How to run Dependabot
+
+Most people are familiar with the Dependabot service that runs on GitHub.com and GitHub Enterprise. Enabling that is as
+simple as [checking a `dependabot.yml` configuration file in to your repository's `.github` directory](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuring-dependabot-version-updates).
+
+However, if you want to run a custom version of Dependabot or run it on another platform, you're not left out in the cold.
+This repo provides the logic necessary for hosting your own standalone Dependabot, as long as you're [not re-selling
+Dependabot to others](TODO:hyper-link-to-readme-license-section). It currently supports opening Pull Requests against repositories hosted on GitHub, Github Enterprise, Azure DevOps, GitLab, BitBucket, and AWS CodeCommit.
+
+Dependabot-Core is a library, so you'll need an entrypoint script of some kind. Here are a few examples to help you get
+started.
+
+>**Note:** If you're looking to run Dependabot locally for development/debugging purposes, see the [Development Guide](TODO:hyperlink-to-readme-section).
+
+## Dependabot-Script
+
+The [dependabot-script][dependabot-script] repo provides a collection of example scripts for configuring the Dependabot
+Core library. It is intended as a starting point for advanced users to run a self-hosted version of Dependabot within their own projects.
+
+## Dependabot CLI
+
+The [Dependabot CLI][dependabot-cli] is a newer tool that may eventually replace `dependabot-script` for standalone use cases.
+While it creates dependency diffs, it's currently missing the logic to turn those diffs into actual PR's. Nevertheless, it
+may be useful for advanced users looking for examples of how to hack on Dependabot.
+
+# Contributing to Dependabot
+
+## Reporting issues and Feature Requests
+
+👋 Want to give us feedback on Dependabot, or contribute to it? That's great - thank you so much!
+
+### Reproducible Example
 
-### Disclosing security issues
+Most bug reports should be accompanied by a link to a public repository that reproduces the problem. Bug reports that
+cannot be reproduced on a public repo using the [dry-run script](TODO:hyperlink) may be closed as "cannot reproduce".
+
+TODO: is there anything else we specifically want here? Generally as long as it's public it's been quick for us to repro,
+as the dry-run script allows specifying exactly which libs to try to bump, so we can specify the failing one to zero in on
+it quickly w/o needing them to manually simplify their manifest...
```

Comment:
@deivid-rodriguez I could use your input here... it seems generally we are good if they provide a public repository link, but if there's anything more specific that we really need please let me know.

I'm planning to hyperlink directly to this section from the issue template, and we'll probably want to add automation that auto-comments with a link to this bullet point when a maintainer applies a label indicating an issue is missing a link to a public repo.

Reply:
I think it's fine. Obviously for fixing purposes and for writing better specs, the simpler a public repro is, the better. But it's hard for users (even for us!) to figure out what can be reduced/simplified so that a bug still reproduces. So I think a public repro is a good compromise.

Reply:
SG, we can always tweak what we ask for down the road...